src/renderer/src/components/agent/AgentsSlice.ts

```diff
@@ -60,14 +60,15 @@ export const saveNewAgent = createAsyncThunk<
 
 export const saveNewArtificialAssistant = createAsyncThunk<
   Agent,
-  { description: string; instructions: string }
->('agents/artificial/assistants/saveNew', async ({ description, name }) => {
-  const response = await IpcService.send<{ agent: any }>(
+  { description: string; name: string; avatarURL: string; }
+>('agents/artificial/assistants/saveNew', async ({ description, name, avatarURL }) => {
+  const response = await IpcService.send<{ agent: Agent }>(
     POST_ONE_ARTIFICIAL_ASSISTANT_CHANNEL,
     {
       params: {
         description,
-        name,
+        name, 
+        avatarURL,
```

Comment:
`avatarUrl` and add list of social cues (see simulation and participants).

e.g. in `SimulationSlice`